src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@1,400;1,700&family=Roboto:wght@500&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+
+  font-family: inherit;
+  font-size: 12px;
+  text-transform: uppercase;
+  padding: 0 50px;
+  margin: 0;
+
+  background: white;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.link_box {
+  display: flex;
+  align-items: center;
+}
+
+.link_list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.list_item {
+  margin-left: 20px;
+}
+
+.link {
+  text-decoration: none;
+  padding: 25px 0;
+  margin-top: 10px;
+  color: black;
+}
+
+.link:hover {
+  transition: 0.5s;
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  display: block;
+  position: absolute;
+  content: "";
+
+  top: 58px;
```

Comment:
But you just need to place it at the bottom, why do you need to calculate this value? Just set bottom: 0 🙂

Reply:
done